app/views/subscriber/group/_tags.html.haml

```diff
@@ -6,12 +6,25 @@
 %p= t('.only_add_persons_with_tags')
 
 .shown
-  = f.collection_select(:tag_list,
-             @possible_tags,
-             :second,
-             :first,
-             { },
-             { class: 'chosen-select',
-               multiple: true,
-               data: { chosen_no_results: t('global.chosen_no_results'),
-                       placeholder: ' ' } })
+  = f.collection_select(:included_subscription_tags,
+                        @possible_tags,
+                        :third,
```

Comment:
I still don't like :third here ...

deluxe version would be to have getters for id and name.

maybe it's to extensive ... but how about creating a TagDecorator with #label method which does the translation ?
we could probably remove the whole Translator class ?

ActsAsTaggableOn::Tag.all.order(:name).decorate ?

I think it's a good idea for you to check this option and get some knowledge about decorators in general.

Reply:
how about my last comment ?